extensions/src/main/java/com/redhat/lightblue/extensions/sequance/Sequance.java

```diff
@@ -0,0 +1,25 @@
+package com.redhat.lightblue.extensions.sequance;
+
+import com.redhat.lightblue.metadata.SimpleField;
+
+/**
+ * TODO: Does SimpleField have all the info required by the backend to manage sequences?
+ *
+ * @author mpatercz
+ *
+ */
+public interface Sequance {
+
+    public void createSequance(SimpleField field);
+
+    public void removeSequance(SimpleField field);
+
```

Comment:
It's for creating and removing sequences using api rather than via metadata. Could be useful for plugins, etc. Or not?